manual/dotnet4.7.2-deu/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,18 @@
+﻿$packageArgs = @{
```

Comment:
Should add a check if .NET 4.7.2 is actually installed to avoid installation on newer .NET versions (see [example](https://github.com/jberezanski/ChocolateyPackages/blob/master/netfx-4.7.2-devpack/tools/helpers.ps1#L24-L28))